subiquity/server/controllers/install.py

```diff
@@ -758,8 +751,7 @@ async def run_unattended_upgrades(self, context, policy):
         apt_conf_path = Path(aptdir) / "zzzz-temp-installer-unattended-upgrade"
         apt_conf_path.write_bytes(apt_conf_contents)
         try:
-            self.unattended_upgrades_ctx = context
-            self.unattended_upgrades_cmd = await start_curtin_command(
+            uu_cmd = await start_curtin_command(
```

Comment:
This could be run_curtin_command I think?

Reply:
Good point!